solutions/standard/variables.tf

```diff
@@ -51,6 +51,13 @@ variable "key_protect_instance_name" {
   description = "The name to give the Key Protect instance that will be provisioned by this solution. Only used if not supplying an existing Key Protect or Hyper Protect Crypto Services instance. If a prefix input variable is specified, it's added to the value in the `<prefix>-value` format."
 }
 
+variable "key_protect_plan" {
+  type        = string
+  default     = "tiered-pricing"
+  description = "The service plan of the Key Protect instance that will be provisioned by this solution. Only used if not supplying an existing Key Protect or Hyper Protect Crypto Services instance."
+  # validation performed in root module
```

Comment:
I would add validation to the DA directly. The error the user will get will come directly from the the DA code then. If it came from some other module, the log may be confusing for users in projects